### PR TITLE
Add bollinger strategy to genetic backtesting

### PR DESCRIPTION
**Pull request by station384**
Updates the genetic backtested for the recently added bollinger strategy

### CODE REVIEW

File: scripts/genetic_backtester/darwin.js [hunk 1]
```diff
@@ -240,6 +241,23 @@ let RangeNeuralActivation = () => {
 };
 
 let strategies = {
+  bollinger: {
+    period_length: RangePeriod(1, 60, 'm'),
+    min_periods: Range(1, 20),
```
Comment (dauv) [inline scripts/genetic_backtester/darwin.js:246]: Any specific reason why you chose min_periods between 1-20? All the other strategies use 1-120, 1-200

File: scripts/genetic_backtester/darwin.js [hunk 2]
```diff
@@ -240,6 +241,23 @@ let RangeNeuralActivation = () => {
...
+    markdown_buy_pct: RangeFloat(-1, 5),
+    markup_sell_pct: RangeFloat(-1, 5),
+    order_type: RangeMakerTaker(),
+    sell_stop_pct: Range0(1, 50),
+    buy_stop_pct: Range0(1, 50),
+    profit_stop_enable_pct: Range0(1, 20),
+    profit_stop_pct: Range(1,20),
+
+  // -- strategy
+    bollinger_size: Range(1, 15),
```
Comment (dauv) [inline scripts/genetic_backtester/darwin.js:256]: bollinger_size defaults to 20 in the conf.js. So I'd assume it should be included in the range aswell?

Comment (station384) [reply scripts/genetic_backtester/darwin.js:256]: thank you,   I missed that in my cleanup before check-in.   I originally had it so low as I knew anything higher in my sample data would result in 0 trades.   but you are correct,  it should be +- atleast 100 % to allow the algorithm to evolve it properly.

**Comment by station384** [comment]
Removed to correct defaults for general use.

File: scripts/genetic_backtester/darwin.js (hunk 2 above)
Comment (dauv) [reply scripts/genetic_backtester/darwin.js:256]: No problem. I had actually implemented bollinger to darwin locally, but wasn't sure on what would be the right parameter ranges.

**Pull request closed without merging**